src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,335 @@
+    - `<service-account-id>`: your service account's ID.
+    - `<organization-id>` with the ID you gathered in [Gather Required Information](#gather-required-information).
+    - `<api_key>`: your service account's API key.
+
+To access a folder's clusters, a user or service account must also have the Cluster Administrator, Cluster Creator, or Cluster Operator role on the folder. The role may be granted by inheritance or directly on a cluster.
+
+## List folder contents
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+{{site.data.alerts.callout_success}}
+To discover the entire folder structure, you must list each folder's contents separately.
+{{site.data.alerts.end}}
+
+1. To list the clusters and folders at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+        --url 'https://management-staging.crdb.io/api/v1/folders/root/contents' \
+        --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace `<api-key>` with your service account's API key.
+
+1. To list the clusters and folders contained within another folder, specify its ID instead of `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+        --url 'https://management-staging.crdb.io/api/v1/folders/<folder_id>/contents' \
+        --header 'Authorization: Bearer <api_key>'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of a folder.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+In the API, each folder is referred to by its ID, regardless of where it is located in the hierarchy.
+{{site.data.alerts.end}}
+
+## Create a cluster in a folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Cluster Administrator](authorization.html#cluster-administrator) or [Cluster Creator](authorization.html#cluster-creator)
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+To create a cluster in a folder, set the `parent_id` field to the ID of the folder.
+
+{{site.data.alerts.callout_info}}
+If you do not specify a `parent_id`, or if you set `parent_id` to `root`, the cluster is created at the level of the organization.
+{{site.data.alerts.end}}
+
+For example, to create a {{ site.data.products.serverless }} deployed on GCP in a folder:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+     --url 'https://management-staging.crdb.io/api/v1/clusters' \
+     --header 'Authorization: Bearer <api-key>' \
+     --header 'content-type: application/json' \
+     --data '{"name":"<cluster_name>","provider":"GCP","spec":{"serverless":{"regions":["us-central1"],"spend_limit":0}, "parent_id":"<parent_id>"}}'
+~~~
+
+Replace:
```

Comment:
I had to add a blank line here or the bullet list was broken.

Reply:
https://cockroachlabs.atlassian.net/browse/DOC-8517